asf_heat_pump_suitability/getters/get_datasets.py

```diff
@@ -1,4 +1,7 @@
 import polars as pl
+import pandas as pd
+import geopandas as gpd
+import logging
 from asf_heat_pump_suitability import config
```

Comment:
This appears to be an empty module (with yaml files in) so you can't import it, rather you need to load the yaml from this location.

Reply:
Apologies, I must have broken this somehow. I restarted my kernel and it works fine!